index.html

```diff
@@ -26,49 +26,289 @@
   </head>
   <body>
-    <header>
-      <nav>
-        <a href="#" class="myLogo"> <span>My Logo</span></a>
-        <i class="fa-solid fa-bars fa-xl myBar"></i>
-      </nav>
-    </header>
-    <section>
-      <div class="headline">
-        <h1>
-          I’m Oscar <br />
-          Glad to see <br />
-          you!
-        </h1>
+    <div class="contaner">
+      <header>
+        <nav>
+          <a href="#" class="myLogo"> <span>My Logo</span></a>
+          <i class="fa-solid fa-bars fa-xl myBar"></i>
+        </nav>
+      </header>
+      <section class="section-1">
+        <div class="headline">
+          <h1>
+            I’m Oscar <br />
+            Glad to see <br />
+            you!
+          </h1>
+        </div>
+        <div class="about-me">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <a class="meet">LET'S CONNECT</a>
+        <div class="social-icons">
+          <ul class="social-icons-list">
+            <li>
+              <a href="#"><img src="assests/images/Vector.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assests/images/Vector (1).png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assests/images/Vector (2).png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="assests/images/Vector (3).png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="/assests/images/Linkedin icon.png" alt=""
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </div>
+
+    <section class="section-2">
+      <div class="myWork myWork1">
+        <div class="snapshot-port">
+          <img src="/assests/images/Snapshoot Portfolio.png" alt="" />
+        </div>
+        <div class="left-block">
+          <div class="primary-text">
+            <h1>Tonic</h1>
+          </div>
+
+          <ul class="second-list">
+            <li class="canopy">CANOPY</li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; Back End
+              Dev
+            </li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; 2015
+            </li>
+          </ul>
+          <div class="work-statment">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="see-proj">
+            <a href="" class="lang">html</a>
+            <a href="" class="lang">css</a>
+            <a href="" class="lang js-anc">javaScript</a>
```

Comment:
Kindly, I suggest that when using any "A" tag, indicate the href attribute, if it doesn't contain a link, use "#".